### PR TITLE
For mozilla-mobile/fenix#8014 - fix feature-pwa manifests db migration

### PR DESCRIPTION
**Pull request by TitanNano**
We need to increase the db version so room creates a new schema and migrate the schema from the original version to the new one.
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/pwa/build.gradle [hunk 1]
```diff
@@ -12,6 +12,7 @@ android {
     defaultConfig {
         minSdkVersion config.minSdkVersion
         targetSdkVersion config.targetSdkVersion
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```
Comment (NotWoods) [inline components/feature/pwa/build.gradle]: Is this necessary? Most of our tests specify the runner with `RunWith`

Comment (TitanNano) [reply components/feature/pwa/build.gradle]: I'm actually not sure. Will check it out.

Comment (TitanNano) [reply components/feature/pwa/build.gradle]: It seems I can't use `RunWith`: `Type mismatch: inferred type is KClass<AndroidJUnitRunner> but KClass<out Runner!> was expected`. 

I borrowed this from https://github.com/mozilla-mobile/android-components/blob/f7d2a4648162bebc98f852e2f2f9d6bd8b6f0636/components/browser/engine-gecko-nightly/build.gradle#L30 where it's also done like this.

**Comment by TitanNano** [comment]
@NotWoods in my initial patch I reverted schema version 1 to before #5724 and migrated from there. But I think  that would cause issues for people who initially got the new version 1, so I added a version 0 which is the original version 1 and then test the migration against both version 1 to 2 and version 0 to 2. I hope this smoothens things out as good as possible.

**Comment by NotWoods** [comment]
bors r=NotWoods

**Comment by TitanNano** [comment]
@NotWoods whups, there are still two unused imports I missed, can I still push an update for this?

**Comment by NotWoods** [comment]
Yes, pushing will cancel the merge. It won't automerge if linting fails anyways

**Comment by NotWoods** [comment]
bors r=NotWoods

**Comment by NotWoods** [comment]
bors retry

**Pull request merged**